student-work/2023/Jacob-Sacks/Mermaid-flowchart.md

```diff
@@ -0,0 +1,31 @@
+# Morning Routine
+
+```mermaid
+
+flowchart TD;
```

Comment:
You're missing the start and end points

A should be ((Start)) and then relabel everything else

Your first block should be 
B(Wake up.) a rounded rectangle, not a circle

the last block should be

V((End))